plugins/versionpress/bootstrap.php

```diff
@@ -54,6 +54,13 @@
     define('VERSIONPRESS_REQUIRE_API_AUTH', true);
 }
 
+if (!defined('VP_COMMIT_MESSAGE_PREFIX')) {
```

Comment:
Is it just semantic distinguishing or is there also mechanism which handles accessibility of those constants ? I have no problem with making it internal. Just asking :)